feat: LEAP-1354: API for Bulk Annotation

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/10682447734) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-sdk/pull/298

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10682447734

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10684856819

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10740677831

triklozoid commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10771229331)
> Successfully merged:  245 files changed, 2306 insertions(+), 2033 deletions(-)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1354'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10771229331

triklozoid committed:
Update poetry.lock

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10774149406

triklozoid commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10775643055)
> Successfully merged:  create mode 100644 docs/source/tutorials/yolo.png

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1354'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10775643055
